backend/review/urls.py

```diff
@@ -2,5 +2,5 @@
 from . import views
 
 urlpatterns = [
-    path('<int:reviewee_id>', views.user_review, name='user_review'),
+    path('<int:reviewee_id>/', views.user_review, name='user_review'),
```

Comment:
Is there any reason to add `/` at the end of the URL? If there is no reason to add it, please remove it since we are following the convention not to add `/` at the end.